src/main/java/com/launchacademy/reviews/controllers/PizzaStylesApiV1Controller.java

```diff
@@ -63,4 +66,21 @@ public Object getById(@PathVariable Integer id){
       return customError.doesntExists();
     }
   }
+
+  @GetMapping("/{id}/{sortOption}")
+  public Object getByIdAndSortReviews(@PathVariable Integer id, @PathVariable String sortOption){
+    Map<String, PizzaStyle> map = new HashMap<>();
+    if(pizzaStyleService.findById(id).isPresent()){
+      PizzaStyle pizzaStyle = pizzaStyleService.findById(id).get();
```

Comment:
Let's set `pizzaStyleService.findById(id)` to a variable before line 73, to avoid calling the findById method twice.  Then we can just check if it is present or not empty and get the value within the optional object.